doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -20,7 +19,7 @@ Procedure
 **********
 The procedure to install CORTX on OVA is mentioned below.
 
-#. Download and unzip the `cortx-va-1.0.2.zip <https://github.com/Seagate/cortx/releases/download/VA/cortx-va-1.0.2.zip>`_ file from `our release page <https://github.com/Seagate/cortx/releases/tag/VA>`_. This contains the virtual machine image.
+#. Download `cortx-va-1.0.3.ova <https://github.com/Seagate/cortx/releases/download/ova-1.0.3/cortx-va-1.0.3.ova>`_ file from `our release page <https://github.com/Seagate/cortx/releases/latest>`_. This contains the virtual machine image.
```

Comment:
Previously @johnbent mentioned to keep all our Releases as pre-release and add the text mentioned in https://github.com/Seagate/cortx/releases/tag/VA
Can we update https://github.com/Seagate/cortx/releases/tag/ova-1.0.3 accordingly or if are planning to publish it as Prod Release.

Reply:
Note sure about doc.sig as its coming from https://github.com/Seagate/cortx/blob/1031200af6e0a947bec09aa028077bb1b31662f2/doc.sig commit

Updated https://github.com/Seagate/cortx/releases/tag/ova-1.0.3 with text and pre-release for now